examples/credentials.js

```diff
@@ -3,16 +3,18 @@
 //
 //   env TRANSLOADIT_KEY=xxx TRANSLOADIT_SECRET=yyy node template_api.js
 //
-// You'll likely just want to `require('transloadit')`, but we're requiring the local
-// variant here for easier testing:
-const Transloadit = require('../src/Transloadit')
+// You may need to build the project first using:
+//
+//   yarn prepack
+//
+const { Transloadit } = require('transloadit')
 
 const transloadit = new Transloadit({
-  authKey: process.env.TRANSLOADIT_KEY,
-  authSecret: process.env.TRANSLOADIT_SECRET,
-  // authKey   : process.env.API2_SYSTEMTEST_AUTH_KEY,
-  // authSecret: process.env.API2_SYSTEMTEST_SECRET_KEY,
-  // endpoint  : 'https://api2-vbox.transloadit.com',
+  authKey: /** @type {string} */ (process.env.TRANSLOADIT_KEY),
+  authSecret: /** @type {string} */ (process.env.TRANSLOADIT_SECRET),
+  // authKey: /** @type {string} */ (process.env.API2_SYSTEMTEST_AUTH_KEY),
+  // authSecret: /** @type {string} */ (process.env.API2_SYSTEMTEST_SECRET_KEY),
+  // endpoint: /** @type {string} */ ('https://api2-vbox.transloadit.com'),
```

Comment:
The three values that are commented out are the result from internal from other repositories and don't serve a purpose for our users. Should we just remove them here?

Reply:
I kept it as-is, but I agree this should probably be removed.